stable/cert-manager/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: cert-manager
+version: 0.2.1
+appVersion: 0.2.3
+description: A Helm chart for cert-manager
+home: https://github.com/jetstack/cert-manager
+keywords:
+  - cert-manager
+  - kube-lego
+  - letsencrypt
+  - tls
+sources:
+  - https://github.com/jetstack/cert-manager
```

Comment:
Can you please add at least one maintainer using their github username. The github username is so that we have a method to contact someone in pull requests and issues.

If the maintainer(s) is already a member of the kubernetes github org consider adding an OWNERS file.

Reply:
Ack